libraries/AP_Scripting/tests/rngfnd_quality_test.lua

```diff
@@ -0,0 +1,209 @@
+
+
+
+-- The UPDATE_PERIOD_MS is the time between when a distance is set and
```

Comment:
Could use a Lua block comment here --[[

Reply:
O - yea, forgot about them. C++ has block comments and line comments and I don't see many block comments these days. I wonder why.